packages/vitest/src/node/core.ts

```diff
@@ -351,35 +351,12 @@ export class Vitest {
   }
 
   async exit() {
-    let timeout: NodeJS.Timeout
-    let resolveClose: () => void
-    let resolveTimeout: () => void
-    const closePromise = new Promise((resolve, reject) => {
-      resolveClose = () => resolve(true)
-      this.close().then(resolve, reject)
-    })
-    // make sure all promises are resolved before exiting process
-    // https://github.com/nodejs/node/blob/master/src/node_file-inl.h#L160
-    const cleanup = () => {
-      clearTimeout(timeout)
-      resolveTimeout?.()
-      resolveClose?.()
-    }
-    const timeoutPromise = new Promise((resolve, reject) => {
-      resolveTimeout = () => resolve(true)
-      timeout = setTimeout(() => reject(new Error(`close timed out after ${CLOSE_TIMEOUT}ms`)), CLOSE_TIMEOUT).unref()
-    })
-    Promise.race([closePromise, timeoutPromise]).then(
-      () => {
-        cleanup()
-        process.exit()
-      },
-      (err) => {
-        cleanup()
-        console.error('error during close', err)
-        process.exit(1)
-      },
-    )
+    setTimeout(() => {
+      console.warn(`close timed out after ${CLOSE_TIMEOUT}ms`)
+      process.exit()
```

Comment:
shouldn't exitCode be 1?

Reply:
I'm not sure. It does not mean that tests have failed. We weren't returning 1 on timeout before this.

Reply:
But it probably does not matter (as this should not happen) and we can have 1 just in case. This way if we have a task that does not finish in time we will get an error and can investigate.

Reply:
We actually have some long running stuff and it fails. Need to investigate